### PR TITLE
Add Integration tests for sublanes using the start process and await end event method

### PR DESCRIPTION
**Pull request by robpal94**
## What did you change?
This PR adds Integration tests for sublanes, using the `startProcessInstanceAndAwaitEndEvent()` Method to start a process instance.

## How can others test the changes?
run `npm test` in the _integration_tests directory. 

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

**Comment by robpal94** [comment]
@ElRaptorus I think we should keep this pr on hold, until https://github.com/process-engine/consumer_api_meta/pull/19 is merged.

File: _integration_tests/bpmn/test_consumer_api_sublane_process.bpmn [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression"><![CDATA[token.current.test_config === 'same_lane']]></bpmn:conditionExpression>
     </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="SequenceFlow_07n5it8" sourceRef="Task_0ukwbko" targetRef="EndEvent_0sgcgmk" />
-    <bpmn:scriptTask id="Task_0ukwbko" name="Write &#34;Hello Lane A&#34; to the console">
+    <bpmn:sequenceFlow id="SequenceFlow_07n5it8" sourceRef="Task_0ukwbko" targetRef="EndEvent_1" />
+    <bpmn:scriptTask id="Task_0ukwbko" name="Write &#34;Hello Lane D&#34; to the console">
       <bpmn:incoming>SequenceFlow_1wy0fnk</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_07n5it8</bpmn:outgoing>
-      <bpmn:script>console.log('Hello Lane A");</bpmn:script>
+      <bpmn:script><![CDATA[console.log('Hello Lane D');]]></bpmn:script>
```
Comment (5ebastianMeier) [inline _integration_tests/bpmn/test_consumer_api_sublane_process.bpmn]: Do you have any idea how all those `CDATA` tags found their way into the changes? Did you switch editors (or versions) or manually add them?

Comment (robpal94) [reply _integration_tests/bpmn/test_consumer_api_sublane_process.bpmn]: Nope, for modelling the Process I used the Camunda Modeller 1.13.1.

Comment (robpal94) [reply _integration_tests/bpmn/test_consumer_api_sublane_process.bpmn]: Just for the record: This happens if you create a Process Definition File with a newer version of the camunda modeller and open this with an older version.

Comment (ElRaptorus) [reply _integration_tests/bpmn/test_consumer_api_sublane_process.bpmn]: Not entirely accurate.
This is not an issue of using different versions of camunda, but using Camunda 1.10 in general.
That version will generate these `CDATA` tags, regardless of where the process file came from (I checked).

**Comment by ElRaptorus** [comment]
@robpal94 Please merge the develop branch now and test these changes again.

**Pull request merged**